bin/load_marker_gene.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# This script takes the marker_gene files (normally available on
+# staging) for a gxa experiment and does the following steps:
+# - Transforms those files into a long (melted) table of experiment id,
+#   cell/run id, and expression. This takes care of avoiding large chunks of
+#   data being kept in memory for long, at the expense of writing to disk (too)
+#   often.
+# - Deletes data from gxa_marker_gene table on postgres if exists 
+# - Loads data into the gxa_marker_gene table.
+# 
+set -e
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]:-$0}" )" &> /dev/null && pwd )
+
+postgres_scripts_dir="${SCRIPT_DIR}/../postgres_routines"
+
+dbConnection=${dbConnection:-$1}
+EXP_ID=${EXP_ID:-$2}
+
+# Check that necessary environment variables are defined.
+[ -z ${dbConnection+x} ] && echo "Env var dbConnection for the database connection needs to be defined. This includes the database name." && exit 1
+[ -z ${EXP_ID+x} ] && echo "Env var EXP_ID for the id/accession of the experiment needs to be defined." && exit 1
+
+# Check that files are in place.
+marker_genes_path=$ATLAS_PROD/analysis/baseline/*/experiments/${EXP_ID}/${EXP_ID}_marker_gene.tsv
```

Comment:
@pmb59 Please sync file name.. change it here or change in R script

Reply:
For proteomics, please add `metric` in file name, it will be easier to accommodate in script